doc/source/security/vault.rst

```diff
@@ -1,4 +1,4 @@
-.. _vault:
+https://github.com/percona/percona-server/pull/3806.. _vault:
```

Comment:
Is this pull request link intentional here?

Reply:
No. Miscopied. Fixed.